pymc3/tests/test_gp.py

```diff
@@ -683,6 +683,67 @@ def testLatent2(self):
         npt.assert_allclose(latent_logp, self.logp, atol=5)
 
 
+class TestLatentVsLatentSparse(object):
+    R"""
+    Compare logp of models Latent and LatentSparse.
+    Should be nearly equal when inducing points are same as inputs.
+    """
+    def setup_method(self):
+        X = np.random.randn(50,3)
+        y = np.random.randn(50)*0.01
+        Xnew = np.random.randn(60, 3)
+        pnew = np.random.randn(60)*0.01
+        with pm.Model() as model:
+            cov_func = pm.gp.cov.ExpQuad(3, [0.1, 0.2, 0.3])
```

Comment:
To make this test stable, could use a covariance function that has a white noise term.  
    pm.gp.cov.ExpQuad(...) + pm.gp.cov.WhiteNoise(0.01)
or something

Reply:
Should these be subclassing `SeededTest` anyways, to make them reproducible?

Reply:
oh yes thats true